sys/auto_init/auto_init.c

```diff
@@ -72,6 +72,10 @@
 #include "lwip.h"
 #endif
 
+#ifdef MODULE_OPENTHREAD
```

Comment:
Ah yes, sorry, I got confused with other auto init radio drivers that doesn't require the include.